libs/documentation/src/lib/components/popup/demos/tooltip/tooltip-basic.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  templateUrl: './tooltip-basic.component.html',
```

Comment:
For stackblitz integration, can you add a selector to this component? The stackblitz app that gets generated has an app.component file that will, in the html, refer to the demo component through its selector.